strawberry/schema/schema.py

```diff
@@ -167,15 +163,14 @@ def execute_sync(
         context_value: Optional[Any] = None,
         root_value: Optional[Any] = None,
         operation_name: Optional[str] = None,
-        validate_queries: bool = True,
-        validation_rules: Optional[Collection[Type[ValidationRule]]] = None,
     ) -> ExecutionResult:
         execution_context = ExecutionContext(
             query=query,
             context=context_value,
             root_value=root_value,
             variables=variable_values,
             operation_name=operation_name,
+            graphql_schema=self._schema,
```

Comment:
do we pass the schema only for caching purposes, right?

Reply:
Yes for caching but also so that it can be used by extensions. Seems like something that would be useful.

Reply:
shall we pass our schema object then?

Reply:
Yeah that makes more sense. Will change